### PR TITLE
As a platform admin, when I visit the admin dashboard, I can create a category. This involves moving categories#create to Admin categories controller

### PR DESCRIPTION
**Pull request by mikedao**
Implemented platform admin abilities.

Admin categories page now has a list of categories, and a platform admin can now add a category from that page.

Removed a platform admin test file that did not work, and duplicated skipped tests.

### CODE REVIEW

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -3,4 +3,30 @@ class Admin::CategoriesController < Admin::BaseController
...
+
+  def category_create
+    @category.save
+    flash[:notice] = "Category Created"
+    redirect_to admin_categories_path
+  end
+
+  def invalid_category
+    flash[:errors] = "Please Try Again"
+    redirect_to admin_categories_path
```
Comment (EmilyMB) [inline app/controllers/admin/categories_controller.rb]: If we are redirect_to admin_categories_path in both branches of the if statement, why not put it outside (line 14)?

File: config/routes.rb [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
   namespace :admin do
     get "/dashboard", to: "base#dashboard"
+    resources :categories
```
Comment (EmilyMB) [inline config/routes.rb]: Maybe we want to limit this to needed routes? From the categories controller, it looks like you are only hitting index and create?

File: test/integration/admin_login_test.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@ class AdminUserTest < ActionDispatch::IntegrationTest
     assert page.has_content?("Platform Admin")
```
Comment (EmilyMB) [inline test/integration/admin_login_test.rb]: Maybe want to change this to assert_equal admin_dashboard_path, current_path? Platform admin should probably be visible from all admin pages, so this test doesn't ensure that upon login the admin lands on the right page.

**Comment by EmilyMB** [comment]
Looks good, @mikedao! A few nitpicks provided above :rainbow:

**Comment by novohispano** [comment]
This involves moving categories#create to Admin categories controller.

**Pull request merged**